assets/lang/sklmessages_pl_PL.properties

```diff
@@ -8,8 +8,10 @@ lang.code=pl_PL
 tab.news=Aktualno\u015bci
 tab.log=Dziennik launchera
 tab.output=Dziennik gry (%s)
-tab.crash=Raport awarii (%s)
+tab.crash=Raport crasha (%s)
```

Comment:
What do you think about changing "awaria" to "crash" also in line 98? On the other hand the official Minecraft launcher uses "awaria" instead of "crash":

![view-crash-report-translation](https://user-images.githubusercontent.com/34160838/41053601-5590d9ee-69bc-11e8-9255-c312184d0d6c.png)

Reply:
Hmm, I think we can use "Ups, wygląda na to, że gra się scrashowała!". What do you think?

Reply:
It sounds nice. Let's do this :)

Reply:
Also, look at this. https://i.imgur.com/hjUntz9.png
Minecraft uses that kind of words because some people (kids below 10y/o)  might not understand the word "crash". But here, we can use it since the age of people who use the launcher is higher, don't you think?

Reply:
"Crash" is a proper word in IT, so yes, we can use it. No one should have anything against that.